CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 0.1.4
+- Comments config now has a style and length sub keys - `style: doxygen(default) | any`, `length: brief(default) | full`, and can be disabled by passing `comments: false`.
```

Comment:
Is there any reason to not make `full` the default? (It is very nice to know all individual parameters when coding against the generated bindings in Dart.)

Reply:
We can, but then the comment is not going to be wrapped.

Reply:
Changing it to full then.

Reply:
I thought we had manual wrapping for doxygen comments?

Reply:
The wrapping is only done for brief comments.
For full comments, We just remove the raw comment markups from every line.
This is because the simple wrapping we use can distort code samples.

Reply:
Ah yeah, of course. That's fine. If the source has proper wrapping, then the generated files should also be okay. (Maybe a couple of characters over the 80 char limit, but that's worth it to preserve to good formatting in my opinion.)